### ISSUE TITLE
Issue with audio attachments 

### ISSUE THREAD

**Issue by alfindlay** [clonemeagain/attachment_preview#50]
Hi, I'm new to osTicket and this plugin so please forgive my poor terminology. 

I'm not sure if I've installed this incorrectly or if this is a bug or browser limitation.  I have plugin v1.11 with osTicket v1.14.3. 
Apache MySQL 10.4.14 PHP 7.4.9

The preview works 100% fine with pdf and img files showing the preview in the ticket thread, but I cannot get it to play wav attachments. the player is visible but greyed out.
![image](https://user-images.githubusercontent.com/4929121/97079783-886a4e00-15ee-11eb-80b1-07b0884d290a.png)

Any ideas, thanks in advance, 
Al

**Comment by clonemeagain** [comment]
Can you screenshot the plugins config? 

Might be time to upgrade this plugin.

**Comment by alfindlay** [comment]
Hi, no worries:
![image](https://user-images.githubusercontent.com/4929121/97093581-1f61f500-1645-11eb-8f28-56b1c919a76b.png)

and 

![image](https://user-images.githubusercontent.com/4929121/97093592-399bd300-1645-11eb-8f8c-95984acb9d23.png)

do you think it might be a compatibility issue?  it doesn't seem to work on any browser  although IE gives slightly more info:
![image](https://user-images.githubusercontent.com/4929121/97093630-8da6b780-1645-11eb-924d-cdea71b29216.png)

**Comment by clonemeagain** [comment]
Yeah, I'm thinking they changed the file API again.. it's not the first time.

**Comment by alfindlay** [comment]
understood.
although it's weird that images seem to work fine and get pulled out of the DB into the cache.  PDF also works fine.

**Comment by ait1ispring** [comment]
It still has the same issue, The preview works 100% fine with pdf and img files showing the preview in the ticket thread, but I cannot get it to play wav attachments. the player is visible but greyed out.

**Comment by clonemeagain** [comment]
Plugin updated to support 1.18.

**Comment by alfindlay** [comment]
Hi Aaron,  thanks for the update.  I've finally managed to update my installation to v1.18.2 and have installed v1.2 of the plugin.

<img width="804" height="291" alt="Image" src="https://github.com/user-attachments/assets/e71adf75-663d-4403-8b1e-a60142d1a9fe" />

The PDF works great again, there are the odd few not previewing but most are working fine.

sadly wav files are still not playing though, the play button remains greyed out:

<img width="1470" height="540" alt="Image" src="https://github.com/user-attachments/assets/0eaf8a81-8c10-45dd-8d08-abd7f87f2135" />

here are my settings:

<img width="955" height="1488" alt="Image" src="https://github.com/user-attachments/assets/34fcd165-ad72-4bfa-ba5e-21dc84e7ca3d" />

great work on the update!
cheers, Al
